controllers/users/helpers.js

```diff
@@ -1,6 +1,30 @@
 import { User } from "#models/User.js";
+import { uploadMiddleware, isImageAndTransform } from "#middlewares/multer.js";
 
 export const findUser = (email) => User.findOne({ email }).lean();
 
 export const signupUser = ({ password, email }) =>
   User.create({ password, email });
+
+export const updateAvatar = async (req, res, id) => {
+  uploadMiddleware.single("avatar")(req, {}, async function (err) {
```

Comment:
Tutaj też bardzo pracować nad jednym stylem na przykład jeśli używasz w ogóle w projekcie arrow function to jak nie ma powodów żeby robić inaczej to trzymać się tej reguły. :)

Reply:
Poprawione